components/CheckInOutForm/index.tsx

```diff
@@ -2,21 +2,32 @@ import { Autocomplete, Box, FormControl, TextField } from '@mui/material'
 import { DateTimePicker } from '@mui/x-date-pickers'
 import dayjs, { Dayjs } from 'dayjs'
 import React from 'react'
-import { ItemDefinition, User } from 'utils/types'
+import { Attribute, AttributeRequest, ItemDefinition, User } from 'utils/types'
```

Comment:
Shouldn't this be InventoryItemAttributeRequest? In VSCode, this gives an error for not being an export of utils/types

Reply:
yea just noticed this as i was going over it w @YarDog4, good catch!